src/interop.rs

```diff
@@ -70,7 +70,7 @@ impl IntoTonicStatus for rusqlite::Error {
     fn into_tonic_status(self) -> Status {
         use rusqlite::ffi::ErrorCode;
         let mut tonic_err = match &self {
-            Self::SqliteFailure(err, _) => match err.code {
+            Self::SqliteFailure(err, _msg) => match err.code {
```

Comment:
```suggestion
            Self::SqliteFailure(err, msg) => match err.code {
```

Starting a variable name with an underscore implies that it is unused, as it silences that lint.

Reply:
Oh, I did this because the message is unused other types, but it doesn't matter.
Adjusted the suggestion. Thanks!